### DISCUSSION TITLE
How to add the animation effect of seawater fluctuation to the tile of the fasttilemap of axmol?

### DISCUSSION THREAD

**Discussion by jcy1001** [axmolengine/axmol#1364] [General]
![OK45OB{(F`J`7}6~7(6 12F](https://github.com/axmolengine/axmol/assets/24226340/979db07a-118f-4500-be0f-8ea3c2e8bfa2)
how do i  make my ocean tile more real like

**Comment by jcy1001** [comment]
The tile-based frame animation effect is very poor

**Reply by DelinWorks** [reply]
Look in the cpp-tests for node effects
